.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: glvd
+
+on:
+  pull_request:
+  release:
+    types:
+    - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and export
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: Containerfile
+          tags: glvd:latest
+          outputs: type=docker,dest=/tmp/glvd.tar
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: glvd
+          path: /tmp/glvd.tar
+          retention-days: 7
+
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+
+    env:
+      SQLALCHEMY_DATABASE_URI: postgres://user:password@localhost:5432/glvd_test
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_DB: glvd_test
+          POSTGRES_USER: user
+          POSTGRES_PASSWORD: password
+        ports:
+        - 5432:5432
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: glvd
+          path: /tmp
+
+      - name: Load image
+        run: |
+          docker load --input /tmp/glvd.tar
+
+      - name: Run image
+        run: |
+          docker run \
+            --env DATABASE_URL="$DATABASE_URL" \
+            --network host \
+            --rm \
+            --user root \
+            glvd:latest \
+            bash -c 'apt update && 
+            apt install -y ca-certificates python3-pytest python3-pytest-flake8 python3-pytest-asyncio mypy && 
+            py.test-3 --flake8 /usr/local/src && 
```

Comment:
`pytest-flake8` breaks more often then not. Don't use it.

Reply:
Alright, removed it.